core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.ts

```diff
@@ -0,0 +1,52 @@
+import { CommonModule, NgClass, NgIf } from '@angular/common';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { SelectItem } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+
+import { DotPageMode } from '@dotcms/dotcms-models';
+
+/**
+ * This component is responsible to display the tab buttons for the edit page.
+ *
+ * @export
+ * @class DotTabButtonsComponent
+ */
+@Component({
+    selector: 'dot-tab-buttons',
+    standalone: true,
+    imports: [CommonModule, ButtonModule, NgIf, NgClass],
+    templateUrl: './dot-tab-buttons.component.html',
+    styleUrls: ['./dot-tab-buttons.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotTabButtonsComponent {
+    @Output() openMenu = new EventEmitter();
+    @Output() clickOption = new EventEmitter();
+    @Input() mode: DotPageMode;
+    @Input() options: SelectItem[];
+    pageMode = DotPageMode;
```

Comment:
same here, readonly protected, assignation of an enum. And please add `const` to `DotPageMode` too.